src/en/docs/releases.md

```diff
@@ -5,8 +5,270 @@ Notes for indivudual misskey releases.
+### Note
+Please run `npm run clean` before building.
+
+Due to the large scale of migration in this release, migration may take some time for some instances.
+If the migration does not finish, you can delete all **records** in the table starting with `__chart__` (do not delete the table itself) and try again, although the chart information will be reset.
+
+### Improvements
+- Chart engine enhancements @syuilo
+	- Reduced table size
+	- Added number of notes with attachments to notes/instance/perUserNotes chart
+	- Added new entry to activeUsers chart
+	- Added new entry to federation chart
+	- Added apRequest chart
+	- network chart eliminated
+- Client: Enable to see the chart on own instance info page @syuilo
+- Client: Allow manual specification of device type @syuilo
+- Client: updated UI icons @syuilo
+- Client: self-hosting UI icons @syuilo
+- Review NodeInfo user and post count content @xianonn
+
+### Bugfixes
+- Client: Fixed "There is a new note" display remains when switching timeline type @tamaina
+- Client: fix UI size issue @tamaina
+- Client: Setting instance information of notes to always show breaks the timeline @Johann150
+- Client: Fixed an issue where the cursor position was not correct when replying depending on the environment @syuilo
+- Client: Fixed a problem that switching the display range of instances does not work in the control panel users and files @syuilo
+- Client: Fixed no update notification dialog @syuilo
+- Client: Follows/Followers Visibility changes won't be saved unless clicking on an other checkbox @Johann150
+- API: Fix API cast @mei23
+- Add instance favicon where it's missing @solfisher
+- Fixed problems with periodic resync of charts not working @syuilo
+
+## 12.103.1 (2022/02/02)
+
+### Bugfixes
+- Client: Fixed an issue with tooltip display position being incorrect.
+
+## 12.103.0 (2022/02/02)
+
+### Improvements
+- Client: Allow instance information to be reacquired from the coalition instance page.
+
+### Bugfixes
+- Client: Fixed an issue where images are hidden when reactions are updated after displaying NSFW images of a post.
+- Client: Fixed problem with "Clip" page not opening
+- Client: Fixed Trends widget not working
+- Client: Fixed Federation widget not working
+- Client: Fixed emoji picker not opening in reaction settings
+- Client: Fixed issue with inclusion of mentions on DM page
+- Client: hashtag retention field in submission form not working
+- Client: Fixed side view not working
+- Client: ensure that specified users does not get duplicates
+- Add `img-src` and `media-src` directives to `Content-Security-Policy` for files and media proxy
+
+## 12.102.1 (2022/01/27)
+
+### Bugfixes
+- Fixed problem with chat not displaying
+
+## 12.102.0 (2022/01/27)
+
+### Note
+After the update, some custom pictograms may not be displayed. In this case, batch exporting the pictograms from the pictogram management page and then batch importing them again from the control panel will fix the problem.
```

Comment:
I think this would be clearer if you use "emoji" instead of "pictogram". Same for entries below in this release.